2016-05-31 11:47:52.052 +0100	START		Running _Test1 Create todo.script
2016-05-31 11:47:52.052 +0100	Exception		Remote Connection Unavailable	3	on_create_todo	Create_todo.script
2016-05-31 11:47:52.052 +0100	EndTestCase		(Duration:"0.312", Errors:"1", Exceptions:"1", StartTime:"2016-05-31 11:47:52 +0100", Successes:"0", TestCase:"_Test1 Create todo.script", Warnings:"0")
2016-05-31 11:47:52.052 +0100	FAILURE		No Remote Connection click Error - Remote Connection Unavailable -- Execution Time 0:00:00 _Test1 Create todo.script
